EstateReporting.IntegrationTests/Shared/SharedSteps.cs

```diff
@@ -787,6 +787,107 @@ public async Task WhenIGetTheMerchantTransactionsByDateReportForEstateAndMerchan
             });
         }
 
+        [Given(@"I set the date for today to ""(.*)""")]
+        public void GivenISetTheDateForTodayTo(String dateForToday)
+        {
+            if (dateForToday.ToUpper() == "TODAY")
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6346796)